doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,105 @@
+The local AHS device
+====================
+
+The local AHS device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed you immediately have access to the `local Braket AHS device <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-simulator-ahs-local>`_ in PennyLane.
```

Comment:
I mean in `BraketAwsAhsDevice` and `BraketLocalAhsDevice`. Something like a note that all parameters get multiplied by `10**6`

Reply:
I believe it's already noted in all the functions where units are relevant. Sometimes in in-line comments in the code, but since almost all the functions are private and not user-facing, I think that's fine (even preferable, as it keeps the comment close to where its used). They're all private methods.

Reply:
You mean the functions in PennyLane, right? Since in PennyLane itself no conversions are happening and this is something specific to the device, we should make it clear in the device docs as well. Something like a `note` block on the device docs
![image](https://user-images.githubusercontent.com/43949391/233330434-c2a00f43-ffc9-4d08-bb38-0091718957cb.png)

Reply:
> Would be good to link to Braket SDK, ideally with install instructions

Added!